scripts/generate-codex-report.py

```
@@ -0,0 +1,125 @@
+                            "if": hook.get("if"),
+                            "commandHash": hashlib.sha256(
+                                command.encode("utf-8")
+                            ).hexdigest(),
+                        }
+                    )
+    return wiring
+
+
+def build(binary: Path, workspace: Path) -> dict:
+    registry = json.loads(
+        subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)